recipe/meta.yaml

```diff
@@ -24,72 +22,73 @@ source:
-      -DCMAKE_INSTALL_PREFIX="{{ prefix }}"  {{ lc }}
-      -DCMAKE_BUILD_TYPE=Release             {{ lc }}
-      -DBUILD_EXAMPLES=OFF                   {{ lc }}
-      -DBUILD_META_EXAMPLES=OFF              {{ lc }}
-      -DBUNDLE_JSON=OFF                      {{ lc }}
-      -DBUNDLE_NLOPT=OFF                     {{ lc }}
-      -DUSE_SVMLIGHT=OFF                     {{ lc }}
-      -DENABLE_TESTING=OFF                   {{ lc }}
-      -DENABLE_COVERAGE=OFF                  {{ lc }}
-      -DUSE_SWIG_DIRECTORS=ON
-    if errorlevel 1 exit 1  # [win]
-
+    cmake ..                                                  {{ lc }}
+      -G"{{vs}}CMAKE_GENERATOR{{ve}}"                         {{ lc }}
+      -DCMAKE_PREFIX_PATH="{{ prefix }}"                      {{ lc }}
+      -DCMAKE_INSTALL_PREFIX="{{ prefix }}"                   {{ lc }}
+      -DCMAKE_BUILD_TYPE=Release                              {{ lc }}
+      -DBUILD_EXAMPLES=OFF                                    {{ lc }}
+      -DBUILD_META_EXAMPLES=OFF                               {{ lc }}
+      -DBUNDLE_JSON=OFF                                       {{ lc }}
+      -DBUNDLE_NLOPT=OFF                                      {{ lc }}
+      -DUSE_SVMLIGHT=OFF                                      {{ lc }}
+      -DENABLE_TESTING=OFF                                    {{ lc }}
+      -DENABLE_COVERAGE=OFF                                   {{ lc }}
+      -DUSE_SWIG_DIRECTORS=ON                                 {{ lc }}
+      -DLICENSE_GPL_SHOGUN=OFF                                {{ lc }}  # [blas == 'mkl']
+      -DUSE_SVMLIGHT=OFF                                      {{ lc }}  # [blas == 'mkl']
+      -DBLA_VENDOR={{ bla_vendor }}                           {{ lc }}
+      -DLAPACK_LIBRARIES={{ prefix }}/lib/{{ blas_lib }}      {{ lc }}
+      -DBLAS_LIBRARIES={{ prefix }}/lib/{{ blas_lib }}
     cmake --build . --target shogun --config Release -- {{ parbuild }}
-    if errorlevel 1 exit 1  # [win]
-
     cmake --build . --target install --config Release -- {{ parbuild }}
-    if errorlevel 1 exit 1  # [win]
-
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - python
-    - cmake
+    - cmake =3.15
   host:
-    - arpack       # [not win]
+    - arpack
     - bzip2
     - ctags
-    - curl
+    - libcurl
+    - libgomp       # [linux]
     - eigen
     - glpk
     - hdf5
-    - json-c       # [not win]
+    - json-c =0.12
     - libxml2
     - lzo
-    - openblas     # [not win]
-    - libprotobuf  # [not win]  # conda-forge doesn't have shared lib yet
-    - rxcpp 4.0.0
+    - mkl-devel     # [blas == 'mkl']
+    - openblas      # [blas == 'openblas']
+    - libprotobuf
+    - rxcpp 4.1.0
     - snappy
     - xz
     - zlib
   run:
-    - arpack       # [not win]
+    - arpack
     - bzip2
     - ctags
-    - curl
+    - libcurl
+    - libgomp       # [linux]
     - glpk
     - hdf5
-    - json-c       # [not win]
+    - json-c =0.12
```

Comment:
Does this need to be pinned so tightly?

Reply:
unfortunately yes :( it has to be somewhere between 0.10... and 0.12. but 0.12 for sure works.

Reply:
Can it be `>=0.10,<=0.12`

Reply:
ok i'll try to figure out the minimum number